src/Sarif/Core/ArtifactLocation.cs

```diff
@@ -106,5 +106,30 @@ public ArtifactLocation Resolve(Run run)
                 ? run.Artifacts[Index].Location
                 : this;
         }
+
+        /// <summary>
```

Comment:
>/// [](start = 8, length = 3)

This was an extension method, but it doesn't need to be because we have partial clases.

Reply:
Right but does this functionality conceptually belong as part of ArtifactLocation? I'm not sure there's a good case for it.

---
In reply to: [472406181](https://github.com/microsoft/sarif-sdk/pull/2035#discussion_r472406181) [](ancestors = 472406181)